kiwipy/rmq/defaults.py

```diff
@@ -2,6 +2,7 @@
 import yaml
 
 # Times are in milliseconds
+MILLI_TO_SECONDS = 1000
```

Comment:
would rather be `SECONDS_TO_MILLISECONDS` no?

Reply:
Too right.